### PR TITLE
Psp 6898 correct document list view paging, use built in table paging.

### PR DESCRIPTION
**Pull request by devinleighsmith**

### CODE REVIEW

File: source/backend/api/Areas/Reports/Controllers/AcquisitionController.cs [hunk 1]
```diff
@@ -81,6 +81,11 @@ public IActionResult ExportAgreements([FromBody] AcquisitionReportFilterModel fi
             }
 
             var agreements = _acquisitionFileService.SearchAgreements(filter);
+            if (agreements is null || !agreements.Any())
+            {
+                // Return 204 "No Content" to signal the frontend that we did not find any matching records.
+                return NoContent();
```
Comment (FuriousLlama) [inline source/backend/api/Areas/Reports/Controllers/AcquisitionController.cs:87]: is this coming from the other pr?

Comment (asanchezr) [reply source/backend/api/Areas/Reports/Controllers/AcquisitionController.cs:87]: @devinleighsmith - this change seems to be coming from a different PR. Could you please rebase this branch?

File: source/frontend/src/features/projects/reports/ProjectExportContainer.test.tsx [hunk 2]
```diff
@@ -96,6 +96,25 @@ describe('ProjectExportContainer component', () => {
...
+    setup();
+    await act(() => viewProps.onExport({ type: 'AGREEMENT' } as Api_ExportProjectFilter));
+
+    const { getAgreementsReport } = useAcquisitionProvider();
+    expect(getAgreementsReport.execute).toHaveBeenCalled();
+    expect(
+      await screen.findByText(/There is no data for the input parameters you entered/i),
+    ).toBeVisible();
+  });
+
```
Comment (asanchezr) [inline source/frontend/src/features/projects/reports/ProjectExportContainer.test.tsx:117]: Looks unrelated to this PR - I think this is coming from the other one

File: source/frontend/src/features/projects/reports/ProjectExportContainer.tsx [hunk 3]
```diff
@@ -41,12 +41,27 @@ export const SideProjectContainer: React.FunctionComponent<ISideProjectContainer
...
+  );
+
+  useEffect(() => {
+    if (getAgreementsReport?.status === 204) {
+      setModalContent({
+        title: 'Warning',
+        message: 'There is no data for the input parameters you entered.',
+        okButtonText: 'Close',
+        closeButton: true,
+        handleOk: () => setDisplayModal(false),
```
Comment (asanchezr) [inline source/frontend/src/features/projects/reports/ProjectExportContainer.tsx:58]: Same comment here about unrelated change

**Review by asanchezr — APPROVED** [review]
Approved but @devinleighsmith I think you need to rebase this branch to get rid of the unrelated changes
